tests/landsat/test_commands.py

```diff
@@ -73,6 +73,12 @@ def test_transform_static_stac_missing_asset_b2_b10(self):
             if item.assets.get("ST_B10.TIF"):
                 item.assets.pop("ST_B10.TIF")
 
+            for asset in item.assets.values():
+                if "geotiff" in asset.media_type:
+                    asset.href = (
+                        "tests/data-files/landsat/"
+                        "LC08_L2SR_081119_20200101_20200823_02_T2_SR_B2_small.TIF"
+                    )
```

Comment:
@gadomski 
Many of my commits were because of the checks which were presenting issues with the re-format. In my point of view, this one as others is not required, it's just formatting.

Reply:
> This one was necessary to add the loop due to the fact that USGS links are not public readable and it was breaking the code.

Right, but I think #105 fixed that issue? Does your PR pass tests w/o these `href` resets?

Reply:
you are right